examples/resources/pingfederate_oauth_client/resource.tf

```diff
@@ -0,0 +1,80 @@
+resource "pingfederate_oauth_client" "simpleOauthClient" {
```

Comment:
Do we need this one in the example?

Reply:
I thought it would be nice if a user just wanted all of the default values and a simple example to have.

Reply:
I think most of our examples have pretty much fully fleshed-out realistic versions of these resources, so imo we don't really need the simple one.